content/library_constants.tex

```diff
@@ -126,10 +126,11 @@
 \vtop{\hbox{\CorCppFor:} 
 \hbox{\hspace*{12mm} \const{SHMEM\_VENDOR\_STRING}}} 
 &
-String representing the vendor name of length less than
-\const{SHMEM\_MAX\_NAME\_LEN}.  In Fortran the string must be \const{SHMEM\_MAX\_NAME\_LEN} 
-and whitespace padded.  It can also be equal in length to \const{SHMEM\_MAX\_NAME\_LEN} 
-since Fortran does not NULL terminate strings. \tabularnewline
+String representing vendor defined information of size at most
+\const{SHMEM\_MAX\_NAME\_LEN}.
+In \CorCpp{}, the string is terminated by a null character.  In \Fortran{}, the
+string of size less than \const{SHMEM\_MAX\_NAME\_LEN} is padded with blank
+characters up to size \const{SHMEM\_MAX\_NAME\_LEN}. \tabularnewline
 \hline
 
 \end{tabular}
```

Comment:
Changes since reading
- Relaxed the semantics of `SHMEM_VENDOR_STRING`. Semantics changed from vendor name to vendor-defined string since it is more useful overall. For vendors that want to provide just a name, this change is fully backwards compatible!